backend/substrapp/tasks/tasks.py

```diff
@@ -66,9 +66,8 @@ def get_algo(subtuple):
     return algo_content
 
 
-def _get_model(model):
-    traintuple_hash = model['traintupleKey']
-    traintuple_metadata = get_object_from_ledger(traintuple_hash, 'queryTraintuple')
+def _get_model(traintuple_hash):
```

Comment:
Mixing `traintuple_hash` and `traintupleKey` argument for the same thing (call of the function and definition of it) is not consistent ! We should chose one name for both.

Reply:
Fixed https://github.com/SubstraFoundation/substra-backend/pull/33/commits/d11f3b4803767841bdc46f22aa87492c46845cfc